actions/check_version/action.yml

```diff
@@ -0,0 +1,8 @@
+name: 'Check version'
+description: 'Check version package'
+outputs:
```

Comment:
@Pham-Gia-Huong 
> description: 'Check version package'
description: 'Check version in podspec'

> version:
     description: "Version package"
version:
     description: "correct podspec version"